lib/kontrast/abstract_runner.rb

```diff
@@ -0,0 +1,32 @@
+module Kontrast
+    class AbstractRunner
```

Comment:
That's a good point, this class was originally created by moving shared code from both runners (hence the name), but it could definitely just be a module that I include in `PageRunner` and `ApiEndpointRunner`